vote_text_drafts/rpip-30.md

```diff
@@ -0,0 +1,57 @@
+> Vote options:
+> - For
+> - Against
+> - Abstain
+
+# RPL Staking Rework (RPIP-30)
+
+This vote aims to align RPL reward spend with rETH capacity created.
+
+### Summary
+- RPL rewards will scale differently, with a focus on aligning RPL rewards with supplying rETH
+- It will be possible to withdraw down to 15% borrowed ETH
+- We will ramp to both of the above states
+- Withdrawing will be a 2-step process where RPL is set to "withdrawing" for 28 days before it can
+  be withdrawn
+
+### Rationale
+The main goal is to align the DAO's spending on RPL rewards with the DAO's goals. In particular, we'll continue to send 70% of inflation to Node Operators, but align it far more with amount of minted rETH.
+
+_Please_ read the Intro document's [context section](https://github.com/Valdorff/rp-thoughts/tree/main/rpl_staking) for a visual representation of spending before and after this change. 
+
+### Detailed Outcomes
+- RPL rewards will scale differently
+  - Adding a minipool will increase RPL rewards (even if you already had enough staked RPL)
+  - Converting an EB16 to two LEB8s will increase rewards (even if you already had enough staked RPL)
+  - You need staked RPL value of ≥10% borrowed ETH to get RPL rewards [unchanged]
```

Comment:
```suggestion
  - You will need staked RPL value of ≥10% borrowed ETH to get RPL rewards [unchanged]
```

Fix tense... Not sure if valid given that its unchanged.

Reply:
going with "will still"